cmd/cloud-controller-manager/app/BUILD

```diff
@@ -24,6 +21,10 @@ go_library(
         "//staging/src/k8s.io/client-go/tools/leaderelection:go_default_library",
         "//staging/src/k8s.io/client-go/tools/leaderelection/resourcelock:go_default_library",
         "//staging/src/k8s.io/cloud-provider:go_default_library",
+        "//staging/src/k8s.io/cloud-provider/controllers/node:go_default_library",
+        "//staging/src/k8s.io/cloud-provider/controllers/nodelifecycle:go_default_library",
```

Comment:
Little worried that the name nodelifecycle here might be confused with https://github.com/kubernetes/kubernetes/tree/master/pkg/controller/nodelifecycle. They should not be confused with each other. cloudnodelifecycle seems cumbersome but something to prevent us crossing the streams would seem prudent.

Reply:
andrewsykim points out that while the above is true, all the actual usage of this controller is done through an alias of CloudNodeLifecycle to disambiguate.